### DISCUSSION TITLE
use scenarios

### DISCUSSION THREAD

**Discussion by carr123** [kornia/kornia#976] [Questions] [answered]
hello,  deep learning using pytorch, for example, cnn, can automatic extract features from an image, and perform classification task.
kornia seems to extract features from an image manually like opencv.
so, i wonder, what is the situation best for using kornia.  could you give me some use cases ？
thanks.

**Comment by shijianjian** [comment]
Have you checked those [tutorials](https://kornia.readthedocs.io/en/latest/tutorials/index.html)? I think you might find some use cases there. Meanwhile, you may also tell us that what you are actually looking for.

**Comment by carr123** [comment, marked as answer]
YES, i checked the tutorial.  still, i don't know when to use kornia .
"Kornia is meant to provide differentiable operators to train nets", is there any blog posts describing this ?
is this way better than oridinary conv nets ?

**Reply by shijianjian** [reply]
In short, if you want to perform cv operations on GPU/TPU or wanted to perform on batched images, you may use kornia. If you have some interesting idea that needs differentiable computer vision like [this](https://arxiv.org/abs/1911.06987) and others, you may use kornia.

**Reply by carr123** [reply]
thanks
